src/Launcher/Launcher.jl

```diff
@@ -508,7 +512,8 @@ Runs the FRG solver. A detailed explanation of the solver parameters is given be
                   bounds[3] gives, in units of the cutoff Λ, the upper bound for the bosonic axis of the two-particle irreducible channels
                   bounds[4] gives, in units of the cutoff Λ, the upper bound for the fermionic axis of the two-particle irreducible channels
 * `max_iter`    : maximum number of parquet iterations
-* `eval`        : initial number of subdivisions for vertex quadrature. Lower number means loss of accuracy, higher will lead to increased runtimes.
+* `min_eval`    : minimum initial number of subdivisions for vertex quadrature. eval is min_eval for parquet iterations.
+* `max_eval`    : maximum initial number of subdivisions for vertex quadrature. eval is ramped up from min_eval to max_eval as a function of the cutoff Λ.
```

Comment:
Maybe explicitely state, that this is only the case for Lambda<1?

Reply:
Will do, seems better.

Reply:
Done!